lib_comfyui/webui/workflow_patcher.py

```diff
@@ -0,0 +1,85 @@
+import functools
+import torch
+import torchvision.transforms.functional as F
+from lib_comfyui import ipc, global_state, default_workflow_types, external_code
+
+
+__original_create_sampler = None
+
+
+@ipc.restrict_to_process('webui')
+def apply_patches():
```

Comment:
```py
def apply_patch():
```
?

I mean, I only see one patch, and I would expect a for loop with that name

Reply:
If we want to add more in the future, this is where we would put them.